### ISSUE TITLE
An error was sent from API Gateway by AWS Lambda.

### ISSUE THREAD

**Issue by fealone** [odd12258053/agraffe#5]
When I deployed to AWS Lambda and request to API Gateway, the following error was sent.

```
[ERROR] KeyError: 'multiValueQueryStringParameters'
Traceback (most recent call last):
  File "/var/task/agraffe/__init__.py", line 57, in _entry_point4aws_lambda
    request={'event': event, 'context': context}
  File "/var/task/agraffe/__init__.py", line 39, in __call__
    cycle(app=self.app)
  File "/var/task/agraffe/services/base.py", line 20, in __call__
    loop.run_until_complete(task)
  File "/var/lang/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/var/task/agraffe/services/base.py", line 38, in run
    await app(self.scope, self.receive, self.send)
  File "/var/task/agraffe/services/aws_lambda.py", line 21, in scope
    query_string = '&'.join(gene_query_string()).encode()
  File "/var/task/agraffe/services/aws_lambda.py", line 16, in gene_query_string
    params = event['multiValueQueryStringParameters'] or 
{}
```

Displayed settings of API gateway in AWS Lambda is below.

```
API endpoint: ****
Authorization: NONE
Method: GET
Resource path: /monitoring
Stage: default
```

The response message by API is below.

```json
{}
```

Do I have the resolve method?

**Comment by fealone** [comment]
I solved it after reading [this](https://aws.amazon.com/jp/blogs/developer/handling-arbitrary-http-requests-in-amazon-api-gateway/).
I think it is easier to understand if it is clearly stated in the document.

Please specify in the document or decide the policy and close this issue.

**Comment by odd12258053** [comment]
@fealone 
Thank you for your comment.
I don't understand the solution.
Please tell me about the solution.
If you encounter [the same issue(4)](https://github.com/odd12258053/agraffe/issues/4), use only the REST API until it has resolved.
I assume that the message `AWS lambda (with API Gateway REST API)` (in `README.md`) is the mean of it.

**Comment by odd12258053** [comment]
Now I released version 0.3.
In version 0.3 and above, agraffe supports the HTTP API.
So, I assume that this issue is solved.
